src/app/retryables-tracker/[address]/PendingRetryables.tsx

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
As is, the fetch doesn't even get reached; the logic inside the PendingRetryables compoenent only executes once when the production site is being built (!) (because it's stateless / propless). With use-client it executes on load.